src/_includes/feature_lists/cloud.njk

```diff
@@ -2,47 +2,75 @@
     set sections = [{
         label: "Features",
         rows: [{
+            id: 'hosting',
```

Comment:
Should making the ID be a filter: https://www.11ty.dev/docs/filters/? 

downcase the label, and replace " " with "-"?

Reply:
Not sure if it makes sense btw